### PR TITLE
level aa contrast plan confirmation

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/186499224

# A brief description of the changes

Current behavior No cucumber, a few contrast issues, and missing tooltip fallback and hover

New behavior: All behaviors improved and cucumber test added

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: CONTRAST_LEVEL_AA_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

Screenshot 2024-01-09 at 3 44 06 PM
Screenshot 2024-01-09 at 3 45 51 PM

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/assets/stylesheets/custom.scss.erb [hunk 1]
```diff
@@ -3652,10 +3652,10 @@ ul.progress-wrapper li.complete {
     list-style: none;
     /*border-left: 2px solid #007bc4;*/
     padding-left: 35px;
-    color: <%= EnrollRegistry[:qle_carousel].settings(:color).item %>;
+    color: var(--theme-primary-blue, <%= EnrollRegistry[:qle_carousel].settings(:color).item %>);
```
Comment (mdkaraman) [inline app/assets/stylesheets/custom.scss.erb:3655]: Let's refactor the call to EnrollRegistry to use the color value directly as a fallback.  The CSS values stored in our Resource Registry .yml files are identical between clients (!!!) and we need to move away from this CSS in RR pattern altogether anyways.

Comment (ATBull81) [reply app/assets/stylesheets/custom.scss.erb]: changed here: [f34b4b2](https://github.com/ideacrew/enroll/pull/3462/commits/f34b4b24c1539f37beae838d28a7522545d8ae4c)

**Review by mdkaraman — APPROVED** [review]
Removes the unnecessary call to EnrollRegistry for the color value.  Adds keyboard navigation and cucumber coverage for contrast standards.  Good work!

**Review by scaustin34 — APPROVED** [review]
LGTM!

**Pull request merged**